Packs/BeyondTrust_Password_Safe/Integrations/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.py

```diff
@@ -394,7 +394,8 @@ def fetch_credentials():
         item = {
             'SystemId': account.get('SystemId'),
             'AccountId': account.get('AccountId'),
-            'DurationMinutes': duration_minutes
+            'DurationMinutes': duration_minutes,
+            'ConflictOption': 'reuse'
```

Comment:
can you elaborate on this change?

Reply:
In some cases when pulling credentials a race condition occurres that resulted in this error in logs: 
```
Error in API call to BeyondSafe Integration [409] - Conflicting request exists. This user or another user has already requested a password for the specified account.) (source: /Users/sbenyakir/dev/go/src/github.com/demisto/server/services/automation/dockercoderunner.go:717)
```
From the docs: 
![Password Safe and BeyondInsight API Guide 2020-11-26 16-59-16](https://user-images.githubusercontent.com/44666568/100366451-80e0ff00-3009-11eb-8e87-d4101f4fc595.png)